flow/connectors/mysql/validate.go

```diff
@@ -1,14 +1,172 @@
 package connmysql
 
+
 import (
 	"context"
+	"database/sql"
```

Comment:
have been avoiding `database/sql`, you can use existing Execute method on MySqlConnector